src/core/fieldformatter/qgsvaluerelationfieldformatter.h

```diff
@@ -51,6 +52,7 @@ class CORE_EXPORT QgsValueRelationFieldFormatter : public QgsFieldFormatter
       QVariant key;
       QString value;
       QString description;
+      QVariant group; //! value used to regroup items during sorting (since QGIS 3.38)
```

Comment:
This needs to be either:

```suggestion
      QVariant group; //!< value used to regroup items during sorting (since QGIS 3.38)
```

or 
```suggestion
      //! value used to regroup items during sorting (since QGIS 3.38)
      QVariant group; 
```